new_project/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "new_project",
```

Comment:
```suggestion
  "name": "new_comit_app",
```

Reply:
We should actually adapt this based on the users input!

Reply:
Yep, I forget to comment on this suggestion, I just wanted something a bit less generic. Adapting on user input sounds good to me.

Reply:
Will make a ticket if you don't mind! :)

Reply:
https://github.com/comit-network/create-comit-app/issues/92